lib/modules/datasource/go/base.ts

```diff
@@ -58,6 +58,16 @@ export class BaseGoDatasource {
       };
     }
 
+    if (goModule.startsWith('code.cloudfoundry.org/')) {
+      const split = goModule.split('/');
```

Comment:
Please add a check to ensure that the split is the expected length

Reply:
Thanks for the quick review! I replaced the usage of `split`, WDYT?